bindings/node/index.d.ts

```diff
@@ -337,13 +337,13 @@ export interface ClientEncryptionEncryptOptions {
    */
   keyAltName?: string;
 
-  /** @experimental */
+  /** @beta Public Technical Preview: The id of the index key. */
```

Comment:
Talked in standup
`@experimental: Public Technical Preview: The id of the index key.`